specification/apimanagement/resource-manager/Microsoft.ApiManagement/2016-10-10/examples/ApiManagementCreateService.json

```diff
@@ -29,8 +29,8 @@
         "location": "West US",
         "etag": "AAAAAAAV8Yo=",
         "properties": {
-            "publisherEmail": "sasolank@microsoft.com",
-            "publisherName": "sasolank@microsoft.com",
+            "publisherEmail": "admin@contoso.com",
```

Comment:
since you touched this, it got my attention: does it make sense that `admin@live.com` was passed, but `admin@contoso.com` comes back? Same for publisher name capitalization.

Reply:
Will fix this.

Reply:
sweet, thanks 🙂